.github/workflows/cicd.yml

```diff
@@ -27,18 +25,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: subosito/flutter-action@v2
         with:
-          flutter-version: ${{env.FLUTTER_MIN_VERSION}}
-      - uses: bluefireteam/melos-action@v3
-      - name: "Analyze with lowest supported version"
-        uses: invertase/github-action-dart-analyzer@v2.0.0
-        with:
-          fatal-infos: true
-
-  analyze-latest:
```

Comment:
this will only work on main so no reason to have two analyzes

Reply:
Maybe better to set it to `if: false`, so that we don't accidentally remove it on a merge later?

Reply:
Or maybe that would actually be even more prone to slip through :sweat_smile:

Reply:
up to you! haha either way wroks for me

Reply:
Not changing anything in the PR is easier, path of least resistance 😄